### PR TITLE
Update ai_item_recommendations.md

### PR DESCRIPTION
**Pull request by sambellbraze**
I've just received this context from the Product Team on what "Item Update frequency" means as it isn't defined anywhere and I think it's very important it is!

### Summary

My PR...

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md [hunk 1]
```diff
@@ -290,11 +290,12 @@ The following table describes the differences between the free and pro version o
...
 | Area                   | Free version                          | Pro version             |
 | :---------------------- | ------------------------------------- | :--------------------------------------- |
-| Item update frequency   | Weekly                                | Daily                                    |
+| Item update frequency<sup>1</sup>   | Weekly                                | Daily                                    |
 | Model retraining frequency  | Monthly                               | Monthly                                   |
-| Maximum recommendations | 1 recommendation per type<sup>1</sup> | 100 recommendations per type<sup>1</sup> |
+| Maximum recommendations | 1 recommendation per type<sup>1</sup> | 100 recommendations per type<sup>2</sup> |
 
-<sup>1. Available recommendation types are AI Personalized, Most recent, and Most popular.</sup>
+<sup>1. This is the frequency that user-specific item recommendations are updated. For example, if a user purchases an item, then it will be used to update their recommendations in the next 'Item update' process</sup>
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md:297]:
```suggestion
<sup>1. This is the frequency at which user-specific item recommendations are updated (all models except Most Popular items, which updates when the model retrains). For example, if a user purchases an item recommended based on AI item recommendations, their recommended items will be updated according to this frequency</sup>
```

File: _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md [hunk 2]
```diff
@@ -290,11 +290,12 @@ The following table describes the differences between the free and pro version o
 
 | Area                   | Free version                          | Pro version             |
 | :---------------------- | ------------------------------------- | :--------------------------------------- |
-| Item update frequency   | Weekly                                | Daily                                    |
+| Item update frequency<sup>1</sup>   | Weekly                                | Daily                                    |
```
Comment (BrandonBrowning) [inline _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md:293]: i would recommend calling this `User update frequency`.  i think that clearly conveys that the most popular information isn't updated, but also helps clarify more specifically that we'll take into account new user event information (which implies AI Personalized and Most Recent being adjusted)

Comment (bre-fitzgerald) [reply _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md:293]: Ah, I missed this feedback. Thanks for reviewing @BrandonBrowning! Will make that change.

Comment (bre-fitzgerald) [inline _docs/_user_guide/sage_ai/recommendations/ai_item_recommendations.md:293]:
```suggestion
| User update frequency<sup>1</sup>   | Weekly                                | Daily                                    |
```

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**